<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowBookEditForm</class>
 <widget class="QWidget" name="BorrowBookEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Borrowing Book Edit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../assets/logo.svg</normaloff>../../assets/logo.svg</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>User ID : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Book ID : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBookID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Borrowed : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Returned : </string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditBorrowed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditReturned">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUseCurrentTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Use current time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReturned">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Returned</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemove">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditUserID</tabstop>
  <tabstop>lineEditBookID</tabstop>
  <tabstop>dateTimeEditBorrowed</tabstop>
  <tabstop>dateTimeEditReturned</tabstop>
  <tabstop>checkBoxReturned</tabstop>
  <tabstop>checkBoxUseCurrentTime</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
